tests/config/pytest_modules.yml

```diff
@@ -82,14 +82,14 @@ angsd/docounts:
   - modules/nf-core/angsd/docounts/**
   - tests/modules/nf-core/angsd/docounts/**
 
-annotsv/installannotations:
```

Comment:
You can avoid committing these chunks by using the `-p` option of `git add`.

Reply:
done!